tests/test_custom_classes.py

```diff
@@ -110,3 +113,29 @@ class CustomGroup(click.Group):
     group = CustomGroup()
     subgroup = group.group()(lambda: None)
     assert type(subgroup) is CustomGroup
+
+
+def test_custom_exception_class():
+    """Subclasses of click.ClickException should be able to override
+    the default message start "Error: {message}" """
+
+    class CustomException(click.ClickException):
+        def format_message(self) -> str:
+            return f"CustomError: {self.message}"
+
+    class MockStdIO(IO, metaclass=ABCMeta):
```

Comment:
This isn't needed, can use `io.StringIO()`

Reply:
I knew there'd be a better way. Lemme try that